fb/app/models/user.rb

```diff
@@ -32,20 +35,6 @@ def mutual_friends(user)
     (friends_v1 + friends_v2) - [user]
   end
 
-  def pending_friends
-    friends1 = friendships.map { |friendship| friendship.friend unless friendship.confirmed }.compact
-    friends2 = inverse_friendships.map { |friendship| friendship.user unless friendship.confirmed }.compact
-    friends1 + friends2
-  end
-
-  def pending_inverse_friends
-    friends2 = inverse_friendships.map { |friendship| friendship.user unless friendship.confirmed }.compact
-  end
-
-  def friend_requests
-    inverse_friendships.map { |friendship| friendship.user unless friendship.confirmed }.compact
-  end
-
   def confirm_friend(requester)
     friendship = inverse_friendships.find { |friendship| friendship.user == requester }
```

Comment:
- [ ] when friendship is confirmed, an additional row with reversed user-friend pair is created.

Reply:
Thank you. I will fix this too.